source/code-generation/query-converter/test-converted-queries/compare-converted-queries.txt

```diff
@@ -1,13 +1,190 @@
+
+      a. From the :guilabel:`Code Generation` tab, click the 
+         :guilabel:`Query Converter` pane.
+
+         .. note::
+
+           If you are not already logged into your Atlas account or have
+           an expired session, you must login to proceed.
+
+      #. On the left-hand :guilabel:`Queries` pane, select a query, view,
+         or stored procedure.
+
+      #. If your query has not been converted, select a 
+         :guilabel:`Target Language` and click :guilabel:`Convert`
+         :icon-lg:`Sparkle`.
+
+   .. step:: Open the :guilabel:`Test Queries` pane
+
+      a. From the :guilabel:`Code Generation` tab, click the 
+         :guilabel:`Query Converter` pane.
+
+      #. Click the :icon-lg:`ChevronUp` button on the 
+         :guilabel:`Test Queries` pane at the bottom-right of 
+         the screen.
+
+   .. step:: (Optional) Specify parameters
+
+      If your converting a stored procedure with parameters, specify the 
+      stored procedure's parameters in the :guilabel:`Source Script` text field.
+
+      For example, for the following PostGreSQL stored procedure:
+
+      .. code-block:: sql
+         :copyable: false
+
+         CREATE OR REPLACE PROCEDURE PUBLIC.CANCEL_CUSTOMER_ORDERS(IN CUST_ID INTEGER)
+         LANGUAGE PLPQSQL
+
+         AS $PROCEDURE$
+         BEGIN 
+            UPDATE ORDERS SET STATUS = 'CANCELLED' WHERE CUSTOMER_ID = CUST_ID;
+         END;$PROCEDURE$
+
+      Specify the ``CUST_ID`` in the :guilabel:`Source Script` text field
+      by replacing ``<value>`` with the customer ID:
+
+      .. code-block:: sql
+         :copyable: false
+         :emphasize-lines: 7
+
+         DO 
+         $$
+            DECLARE
+               CUST_ID INTEGER;
+            BEGIN
+               --TODO: Set parameter values here
+               CUST_ID := <value>
+
+               CALL public.cancel_customer_orders(cust_id);
+            END;
+         $$  
+
+   .. step:: Run the source query
+
+      a. On the :guilabel:`Test Queries`
+         pane, click the :guilabel:`Run Source Query` 
+         button.
+
+      #. Enter the connection details to your source database.
+
+      #. Click :guilabel:`Run` on the 
+         :guilabel:`Connection Details` form.
+
+         The data from the source database populates in 
+         :guilabel:`Results` user interface. You can click 
+         the :guilabel:`Messages` pill to see execution 
+         statistics such as ``Execution Time``, ``Row Count``, 
+         error messages, and any print statements.
+      
+   .. step:: Run the converted query
+
+      a. On the :guilabel:`Test Queries`
+         pane, click :guilabel:`Run Converted Query` 
+         button.
+
+      #. Enter the connection details to your database.
+
+      #. Click :guilabel:`Run` on the 
+         :guilabel:`Connection Details` form.
+
+         The data from the destination database populates 
+         on the right-side of :guilabel:`Results`.
```

Comment:
[suggestion]

Suggesting this for consistency on how this component is referred to through out the doc

```suggestion
         on the right-side of the :guilabel:`Results` pane.
```